From 26ba48105f46df348b27a8e2cc2dbff13c6ceabe Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Tue, 6 Nov 2018 20:05:39 +0200
Subject: [PATCH 94/96] ucm-imx8: dts: Add BT dummy regulator

Add the dummy regulator in order to turn BT always on via the appropriate GPIO.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/ucm-imx8-wifi.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8-wifi.dtsi b/arch/arm64/boot/dts/compulab/ucm-imx8-wifi.dtsi
index a5d0fae..422ad88 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8-wifi.dtsi
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8-wifi.dtsi
@@ -43,6 +43,15 @@
 			enable-active-high;
 			status = "okay";
 		};
+
+		btreg_on: btregon-regulator {
+			compatible = "regulator-fixed";
+			regulator-name = "btreg_on";
+			gpio = <&gpio1 4 0>;
+			enable-active-high;
+			regulator-always-on;
+			status = "okay";
+		};
 	};
 
 };
-- 
1.9.1

